spec/implementation_matrix.md

```diff
@@ -1,63 +1,71 @@
 # Readiness matrix
 
-|                       | [3540] | [3670] | [4200]  | [4750] | [5450] | [6206] | [7480] | [7069] | [7620] | [7698] | [663] |
-|-----------------------|--------|--------|---------|--------|--------|--------|--------|--------|--------|--------|-------|
-| [**Megaspec**]        | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍    |
-| **EIP**               | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍    |
-| **testing**           | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 🚧     | 🚧     | 🚧     |        | 👍    |
-| [besu]                | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍      | 👍   |
-| erigon                | 🚧     | 🚧     | 🚧      | 🚧     | 🚧     | 🚧     | 🚧     |        |        |        | 🚧    |
-| [ethereumjs]          | 🚧     | 🚧     | 🚧      | 🚧     | 🚧     | 🚧     | 🚧     | 🚧     | 🚧     | 🚧     | 🚧    |
-| [evmone]              | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍    |
-| [geth]                | 🚧     | 🚧     | 🚧      | 🚧     | 🚧     | 🚧     | 🚧     | 🚧     | 🚧     |        | 🚧    |
-| [nethermind]          | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍      | 👍   |
-| revm                  | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍    |
-| [solidity POC]        | 👍     | N/A    | 👍      | 👍     | N/A    | 👍     | 👍     | 👍     | 👍     | 👍     |  👍   |
-| [vyper]               | 🚧     | N/A    | 🚧      | 🚧     | N/A    |        |        |        |        |        |       |
-| [EELS]                | 👍     | 👍     | 👍      | 👍     | 👍     | 👍     | 👍     | 👍     | 👍     | 👍      | 👍   |
-
-[3540]: https://eips.ethereum.org/EIPS/eip-3540
-[3670]: https://eips.ethereum.org/EIPS/eip-3670
-[4200]: https://eips.ethereum.org/EIPS/eip-4200
-[4750]: https://eips.ethereum.org/EIPS/eip-4750
-[5450]: https://eips.ethereum.org/EIPS/eip-5450
-[6206]: https://eips.ethereum.org/EIPS/eip-6206
-[7480]: https://eips.ethereum.org/EIPS/eip-7480 
-[7069]: https://eips.ethereum.org/EIPS/eip-7069 
-[7620]: https://eips.ethereum.org/EIPS/eip-7620 
-[7698]: https://eips.ethereum.org/EIPS/eip-7698 
-[663]: https://eips.ethereum.org/EIPS/eip-663
-
-[**Megaspec**]: https://github.com/ipsilon/eof/blob/main/spec/eof.md
+|                | [eof-devnet-0]           | [eof-devnet-1]       | eof-devnet-2 |
+|----------------|--------------------------|----------------------|--------------|
+| Megaspec       | ✅ [Megaspec v0.1.1]     | ✅ [Megaspec main]   | 🚧           |
+| **EIP**        | ✅ [EIP-7692]            | ✅ [EIP-7692]        | 🚧           |
```

Comment:
Link to specific commits

Reply:
linked specifically for devnet-0, but we don't have a specific commit or tag for devnet-1 yet, so will hold off.